### PR TITLE
Make some spelling and grammar corrections to TS.

### PR DESCRIPTION
**Pull request by arswaw**
I hope that this is part of the contributing guides. I don't know if you can just open up a Pull Request to fix this kind of issue.

### CODE REVIEW

File: docs/TypeScript.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 <a id="typescript"></a>
 ## TypeScript
-Fastify is shipped with a typings file, but it still require to install `@types/node`, depending on the Node.js version that you are using.
+Fastify is shipped with a typings file, but you may need to install `@types/node`, depending on the Node.js version you are using.
 
 ## Types support
-We do care about the TypeScript community, but the framework is written in plain JavaScript and currently no one of the core team is a TypeScript user while only one of the collaborators is.
-We do our best to have the typing updated with the latest version of the API, but *it can happen* that the typings are not in sync.<br/>
+We do care about the TypeScript community, but the framework is written in plain JavaScript and currently no one on the core team is a TypeScript user while only one of the collaborators is.
+We do our best to have the typings updated with the latest version of the API, but *it can happen* that the typings are not in sync.<br/>
```
Comment (mcollina) [inline docs/TypeScript.md]: This is not true anymore, we have multiple typescript users in the core team. @EthanArrowood is currently doing most of the work on typescript, and he is reworking all our types.

@fastify/core I propose we change this, removing the sentence about not having typescript core maintainers. Wdyt?

Comment (Eomm) [reply docs/TypeScript.md]: I'm also learning it in order to check simples PR

For what I saw, I would remove this sentence in next months

Comment (delvedor) [reply docs/TypeScript.md]: > I propose we change this, removing the sentence about not having typescript core maintainers. Wdyt?

👍

Comment (Ethan-Arrowood) [reply docs/TypeScript.md]: I agree!

Comment (arswaw) [reply docs/TypeScript.md]: I'm glad I was able to spur such a productive discussion.

Comment (mcollina) [reply docs/TypeScript.md]: Thanks! We needed this discussion, and updating this text, considering the state of things. That’s the benefit of open governance.

Would you mind updating that text to reflect this discussion?

Comment (arswaw) [reply docs/TypeScript.md]: Done. It's in commit 4321eb2.

**Pull request merged**